pkg/config/debug.go

```diff
@@ -27,3 +29,14 @@ type DebugConfig struct {
 
 	Messages *MessagesConfig `toml:"messages" json:"messages"`
 }
+
+// ValidateAndAdjust validates and adjusts the debug configuration
+func (c *DebugConfig) ValidateAndAdjust() error {
```

Comment:
In fact, I would love it to be separate. For example, call it validate() + complete(), because it might be possible to call validate independently to simply verify the configuration. But that's fine, it looks good for now.